components/filter/filter.js

```diff
@@ -180,7 +180,8 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 				display: -webkit-box;
 				hyphens: auto;
 				-webkit-line-clamp: 2;
-				word-break: break-word;
+				overflow-wrap: anywhere;
+				word-break: normal;
```

Comment:
`break-word` has been deprecated but is still supported currently. See [here](https://drafts.csswg.org/css-text/#word-break-property) at the bottom of the section, which states "When specified, this has the same effect as word-break: normal and [overflow-wrap: anywhere](https://drafts.csswg.org/css-text/#propdef-overflow-wrap), regardless of the actual value of the overflow-wrap property." so I've replaced the `word-break` with those.

Reply:
Yeah I've have really good results using `overflow-wrap: anywhere;`.

Do we still need `word-break: normal;` if we're removing `word-break: break-word;`?

Reply:
I think probably not as `normal` is the default. I kept it in to protect against future additions of `word-break` not noticing the `overflow-wrap` but that's likely an edge case issue and it should be safe to remove.